website/src/contribute/index.md

```diff
@@ -45,269 +51,180 @@ There are lots of opportunities:
  - improve your favorite language SDK (Java, Python, Go, etc)
  - improve specific runners (Apache Apex, Apache Flink, Apache Spark, Google
    Cloud Dataflow, etc)
+ - improve or add IO connectors
  - work on the core programming model (what is a Beam pipeline and how does it
    run?)
  - improve the developer experience on Windows
+ - adding answers to the [contribution FAQ]({{ site.baseurl }}/contribute/FAQ)
+ - organize local meetups of users or contributors to the Apache Beam SDK
 
 Most importantly, if you have an idea of how to contribute, then do it!
 
-For a list of open starter tasks, check
-[https://s.apache.org/beam-starter-tasks](https://s.apache.org/beam-starter-tasks).
-
-## Finding Help
+## Prerequisites for contributing code
+
+To contribute code, you need
+ - a GitHub account
+ - a Linux, macOS, or Microsoft Windows development environment with Java JDK 8 installed
+ - [Docker](https://www.docker.com/) installed for some tasks including building worker containers and testing website
+   changes locally
+ - [Go](https://golang.org) 1.10 or later installed for Go SDK development
+ - Python, virtualenv, and tox installed for Python SDK development
+ - for large contributions, a signed [Individual Contributor License
+   Agreement](https://www.apache.org/licenses/icla.pdf) (ICLA) to the Apache
+   Software Foundation (ASF).
+
+## Submitting your first code change
+
+1. Subscribe to the [dev@ mailing list]({{ site.baseurl}}/community/contact-us/).
```

Comment:
I'm not sure I would list this as a step in submitting a code change; this is useful if you plan to be a repeat contributor or want to make some deep changes; but if you have something small I don't believe you should need to make a long-term commitment to the community.

Reply:
Made it softer,

Consider subscribing to the dev@ mailing list, especially if you plan to make more than one change or the change will be large. All decisions happen on the public dev list.